docs/ref/django-admin.txt

```diff
@@ -979,6 +979,20 @@ Lets you pass a command as a string to execute it as Django, like so::
 
     django-admin shell --command="import django; print(django.__version__)"
 
+.. versionadded:: 1.11
+
+Lets you pass a script to execute via standard input on Linux and have it
+output to standard output, ie::
```

Comment:
"Lets you pass a script" is a bit fluffy. Also you mean Unix or Posix, not just Linux 😉  Perhaps something like:

"You can now pass code in on standard input and it will be executed without any extra output from the REPL. For example::"